client/verta/verta/registry/entities/_model.py

```diff
@@ -907,3 +907,21 @@ def delete(self):
         request_url = "{}://{}/api/v1/registry/registered_models/{}".format(self._conn.scheme, self._conn.socket, self.id)
         response = _utils.make_request("DELETE", request_url, self._conn)
         _utils.raise_for_http_error(response)
+
+    def set_actionType(self, actionType):
+        if not actionType:
+            raise ValueError("actionType is not specified")
+        self._update(self.RegisteredModelMessage(action_type=actionType))
+
+    def get_actionType(self):
+        self._refresh_cache()
+        return self._msg.action_type
+
+    def set_dataType(self, dataType):
+        if not dataType:
+            raise ValueError("dataType is not specified")
+        self._update(self.RegisteredModelMessage(data_type=dataType))
+
+    def get_dataType(self):
+        self._refresh_cache()
+        return self._msg.data_type
```

Comment:
Similar to my comment above, could you change these to be similar to lock_level ([`set_lock_level()` and `get_lock_level()`](https://github.com/VertaAI/modeldb/blob/b90a637/client/verta/verta/registry/entities/_modelversion.py#L1200-L1234))?

Reply:
done